WORKSPACE

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+workspace(name = "gnoi")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# -- Load Dependencies ---------------------------------------------------------
+load("gnoi_infra_deps.bzl", "gnoi_infra_deps")
+
+gnoi_infra_deps()
+
+
+load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
+
+grpc_deps()
+
+bind(
```

Comment:
Use of bind is officially discouraged: https://docs.bazel.build/versions/master/be/workspace.html#bind
Is this actually needed?